package.json

```diff
@@ -21,7 +21,9 @@
     "test:server": "jest --config ./jest.server.js",
     "test:report": "yarn test --ci --silent || true",
     "build": "node ./writeConfig.js && webpack --config webpack.config.js",
-    "prepublish": "yarn build"
+    "prepublish": "yarn build",
```

Comment:
nit: This should have been deprecated awhile back. Can we safely update this to `prepare` instead?

Reply:
done